packages/core/src/App/Containers/Modals/trading-assessment-existing-user.jsx

```diff
@@ -0,0 +1,179 @@
+
+        return () => setIsTradingAssessmentForExistingUserEnabled(false);
+    }, []);
+
+    const handleSubmit = async values => {
+        if (values.risk_tolerance === 'No') {
+            setShouldShowTradeAssessmentForm(false);
+            setShouldShowRiskWarningModal(true);
+        } else {
+            const form_payload = {
+                trading_experience_regulated: { ...values },
+            };
+            const response = await setFinancialAndTradingAssessment(form_payload);
+            const { trading_score } = response.set_financial_assessment ?? {};
+            await updateAccountStatus();
+            setShouldShowTradeAssessmentForm(false);
+            if (trading_score === 0) {
+                setShouldShowWarningModal(true);
+            } else {
+                setShouldShowAssessmentCompleteModal(true);
+            }
+        }
+    };
+
+    const handleAcceptAppropriatenessTestWarning = () => {
+        setShouldShowWarningModal(false);
+        if (window.location.href.includes(routes.trading_assessment)) {
+            setShouldShowAssessmentCompleteModal(false);
+        } else {
+            setShouldShowAssessmentCompleteModal(true);
+        }
+    };
+
+    const handleAcceptRisk = () => {
+        setFormValue(prev_state => ({ ...prev_state, risk_tolerance: 'Yes' }));
+        setShouldMoveToNext(true);
+        setShouldShowRiskWarningModal(false);
+        setShouldShowTradeAssessmentForm(true);
+    };
+
+    if (should_show_risk_warning_modal) {
+        return (
+            <RiskToleranceWarningModal
+                show_risk_modal={should_show_risk_warning_modal}
+                title={localize('Risk Tolerance Warning')}
+                button_text={localize('Yes, I understand the risk.')}
+                onClick={handleAcceptRisk}
+                body_content={
+                    <Localize
+                        i18n_default_text='CFDs and other financial instruments come with a high risk of losing money rapidly due to leverage. You should consider whether you understand how CFDs and other financial instruments work and whether you can afford to take the high risk of losing your money. <0/><0/> To continue, you must confirm that you understand your capital is at risk.'
+                        components={[<br key={0} />]}
+                    />
+                }
+            />
+        );
+    } else if (should_show_warning_modal) {
+        return (
+            <TestWarningModal
+                show_risk_modal={should_show_warning_modal}
+                body_content={
+                    <Text as='p' size='xs'>
+                        <Localize
+                            i18n_default_text='In providing our services to you, we are required to ask you for some information to assess if a given product or service is appropriate for you and whether you have the experience and knowledge to understand the risks involved.<0/><0/>'
```

Comment:
please check if it is ok ty implement <0/><0/>

Reply:
This looks fine to me

Reply:
Yea there's two because we wanted a bigger gap in between the text.